doc/setup-commands.rst

```diff
@@ -144,7 +144,7 @@ This has the following effects:
    the set of databases via :option:`--package-db` (and related flags): these
    dependencies are assumed to be up-to-date. A dependency can be
    explicitly specified using :option:`--dependency` simply by giving the name
-   of the internal library; e.g., the dependency for an internal library
+   of the sublibrary; e.g., the dependency for an sublibrary
    named ``foo`` is given as
    ``--dependency=pkg-internal=pkg-1.0-internal-abcd``.
```

Comment:
This does not relate to the change, but is this correct? It looks wrong to me because `foo` does not feature in what is given as the value of the `--dependency` option.

Reply:
I updated it with `--dependency", "Lib:foo=foo-0.1-abc`, lifted from our testuite.